core/management.c

```diff
@@ -55,7 +55,8 @@ coap_status_t handle_dm_request(lwm2m_context_t * contextP,
                                 lwm2m_uri_t * uriP,
                                 void * fromSessionH,
                                 coap_packet_t * message,
-                                coap_packet_t * response)
+                                coap_packet_t * response,
+                                bool* notify_change)
 {
```

Comment:
About this notify_change behavior : IMHO it should always be the object's responsability to trigger notify through lwm2m_resource_value_changed() API. It would prevent having two notifications sent for the same event.